DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj

```
@@ -1,14 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Interesting. What does this do?

---

it will force us to address any warnings in the build. will fail instead of warn.  Happy to remove, but I think it seem like good practice